app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt

```diff
@@ -115,6 +180,109 @@ data class MessageComposerInnerState(
 
         messageComposeInputState = newState
     }
+
+    private fun applyMentionStylesIntoText(text: TextFieldValue): TextFieldValue {
+        // For now there is a known issue in Compose
+        // https://issuetracker.google.com/issues/199768107
+        // It do not allow us to set some custom SpanStyle into "EditableTextView" :(
+        // But maybe someday they'll fix it, so we could use it
+        val spanStyles = mentions.map { mention ->
+            AnnotatedString.Range(mentionSpanStyle, mention.start, mention.start + mention.length)
+        }
+
+//        return text.copy(
+//            annotatedString = AnnotatedString(
+//                text.annotatedString.text,
+//                spanStyles,
+//                text.annotatedString.paragraphStyles
+//            )
+//        )
```

Comment:
Can we remove it ?

Reply:
as i commented above there is an issue in Compose lib that does not allow to change `SpanStyle` for `EditTextView`. 
I kept this code commented so in a future when (if) compose-issue will be fixed we can just uncomment the code, without any additional actions :)